tests/e2e/compareScenarios.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { expect, test } from "@playwright/test";
+import waitForNewScenarioPage from "~/tests/e2e/helpers/waitForNewScenarioPage";
+import checkRApiServer from "./helpers/checkRApiServer";
+import selectParameterOption from "~/tests/e2e/helpers/selectParameterOption";
+import { parameterLabels, scenarioPathMatcher } from "./helpers/constants";
+
+const baselinePathogenOption = "SARS 2004";
+
+test.beforeAll(async () => {
+  checkRApiServer();
+});
+
+test("Can compare multiple scenarios", async ({ page, baseURL }) => {
+  await waitForNewScenarioPage(page, baseURL);
+
+  await selectParameterOption(page, "pathogen", "SARS 2004");
+  await selectParameterOption(page, "response", "Elimination");
+  await selectParameterOption(page, "country", "United States");
+  await page.click(`div[aria-label="${parameterLabels.vaccine}"] label[for="medium"]`);
+  await page.fill(`input[aria-label="${parameterLabels.hospital_capacity}"][type="number"]`, "305000");
+
+  await page.click('button:has-text("Run")');
+
+  await page.waitForURL(new RegExp(`${baseURL}/${scenarioPathMatcher}`));
+  await expect(page.getByText("Simulate a new scenario")).not.toBeVisible();
+
+  // Run a second analysis with a different parameter, using the parameters form on the results page.
```

Comment:
this comment feels like it doesn't belong here...?

Reply:
Oops